### PR TITLE
Breakout sessions proposal

### PR DESCRIPTION
**Pull request by littledan**
We are shaping up to have a lot of time left at the end of the meeting. Should we collect breakout session topics? Some ideas to start with in this PR, but ideas from others are encouraged.

### CODE REVIEW

**Comment by wycats** [comment]
Big +1 from me. I like the proposal ideas!

File: 2018/11.md [hunk 1]
```diff
@@ -137,6 +137,23 @@ Supporting materials includes slides, a link to the proposal repository, a link
     | topic | presenter |
     |-------|-----------|
 
+1. Breakout sessions
+
+    If time permits, we can schedule breakout sessions after the regularly scheduled agenda items. l
```
Comment (Jxck) [inline 2018/11.md]: Typo on `l` before `Let'`

File: 2018/11.md [hunk 2]
```diff
@@ -137,6 +137,23 @@ Supporting materials includes slides, a link to the proposal repository, a link
...
+    Let's draft breakout topics below, but continue to accept new proposals at the physical meeting.
+    
+    | topic | discussion prompt |
+    |-------|-------|
+    | IDL deep dive | Examine the semantics of WebIDL, observe its mismatches and pain points with JS, and think about what we want for IDL in JS |
+    | JS standard library infrastructure | How should built-in modules work? |
+    | JS standard library contents | What should we put in the JS standard library? |
+    | Functional programming in JS | How should JavaScript evolve to better support functional programming? |
+    | WebAssembly/JS interaction | Review various WebAssembly proposals for interacting with JS (gc, host-binfings, modules) and discuss ways to improve interaction |
```
Comment (chicoxyzzy) [inline 2018/11.md]: Also:
- Type Reflection for JS API
- BigInt <-> i64

There is a typo in `host-bindings`

File: 2018/11.md [hunk 3]
```diff
@@ -137,6 +137,23 @@ Supporting materials includes slides, a link to the proposal repository, a link
...
+    | Diversity, inclusion and outreach | What are our next steps to improve further? |
+    | Shared/remote JS code | Can we enable lightweight, ergonomic ways to run JavaScript code in different threads/processes (e.g. Workers)?|
```
Comment (chicoxyzzy) [inline 2018/11.md]: Interesting. Typed Objects proposal potentially could help with this.

**Comment by ljharb** [comment]
We ended up doing breakout sessions; is this still something we should merge?

**Comment by littledan** [comment]
Closing per https://github.com/tc39/agendas/pull/484#issuecomment-443036088

**Pull request closed without merging**